packages/wallets/src/components/DerivAppsSection/DerivAppsGetAccount.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import DerivApps from '../../public/images/deriv-apps.svg';
+import { WalletButton, WalletText } from '../Base';
+
+type TProps = {
+    isMaltaWallet?: boolean;
+};
+
+export const DerivAppsGetAccount: React.FC<TProps> = ({ isMaltaWallet }) => {
```

Comment:
Lets export it at the bottom. Lets follow the same convention around

Reply:
@thisyahlen-deriv ok, but I thought we have to export default at the end of the file and it's not related to name export. But of course, I'll change it soon